HunterPie.Integrations/Datasources/MonsterHunterRise/Entity/Player/MHRPlayer.cs

```diff
@@ -774,20 +773,26 @@ private void GetPlayerWirebugs()
 
         // Update temporary wirebug from Frenzied Bloodlust skill
         bool hasWild = Wirebugs[lastIdx].IsAvailable;
-
-        if (hasWild)
-            lastIdx = ^1;
-
         bool isSkillActive = Process.Memory.Deref<uint>(
             AddressMap.GetAbsolute("ABNORMALITIES_ADDRESS"),
             AddressMap.Get<int[]>("WIREBUG_NUM_FROM_SKILL_OFFSETS")
         ) > 0;
+        MHRWirebugExtrasStructure temporaryData = new();
+
+        if (isSkillActive)
+        {
+            temporaryData = Process.Memory.Deref<MHRWirebugExtrasStructure>(
+                AddressMap.GetAbsolute("ABNORMALITIES_ADDRESS"),
+                AddressMap.Get<int[]>("WIREBUG_EXTRA_DATA_FROM_SKILL_OFFSETS")
+            );
+            temporaryData.Timer /= AbnormalityData.TIMER_MULTIPLIER;
+        }
+        else
+            temporaryData.Timer = 0.0f;
```

Comment:
This can be removed since it's redundant, when you initialize a structure, the default value for a float is already 0